Dockerfile

```diff
@@ -16,10 +16,6 @@ COPY . .
 FROM base as builder
 RUN go build -o conftest -ldflags="-w -s" main.go
 
-## TEST STAGE ##
```

Comment:
What's the rationale for removing this target? It can be useful for running the tests without needing to install a Go environment.

Reply:
In hindsight, it makes more sense to leave it in, at least for now. It felt a little much to have a dedicated stage for just `go test`. I would expect if you're wanting to make changes to Conftest, and test them, requiring Go is a low bar. But the same argument can be made for the `build` stage.

Maybe a better end-state would be a stage for Go builds/tests, and use that artifact for examples/acceptance/release.

Reply:
This stage has been re-added.